Scoop/Controller/Onboarding/OnboardingViewController.swift

```diff
@@ -9,34 +9,101 @@ import UIKit
 
 protocol OnboardingDelegate: AnyObject {
     func didTapNext(_ viewController: UIViewController, nextViewController: UIViewController?)
+    func didTapBack(_ viewController: UIViewController, previousViewController: UIViewController?)
 }
 
 class OnboardingViewController: UIViewController {
     
-    internal weak var delegate: OnboardingDelegate?
-    internal var nextAction: UIAction?
+    weak var delegate: OnboardingDelegate?
+    var nextAction: UIAction?
+    let backButton = UIButton()
 
     override func viewDidLoad() {
         super.viewDidLoad()
+        setupBackButton()
         view.backgroundColor = .white
     }
     
     func setupNextButton(action: UIAction) {
-        let trailingButtonMultiplier = 0.14
-        let bottomButtonMultiplier = 0.12
+        let bottomButtonMultiplier = 0.1
         
         let button = UIButton()
         button.setImage(UIImage(named: "nextbutton"), for: .normal)
         button.clipsToBounds = true
         button.addAction(action, for: .touchUpInside)
         view.addSubview(button)
-        
         button.snp.makeConstraints { make in
-
             let screenSize = UIScreen.main.bounds
-            print(screenSize.width)
-            make.trailing.equalTo(view.safeAreaLayoutGuide).inset(screenSize.width * trailingButtonMultiplier)
+            make.trailing.equalTo(view.safeAreaLayoutGuide).inset(20)
             make.bottom.equalTo(view.safeAreaLayoutGuide).inset(screenSize.height * bottomButtonMultiplier)
         }
     }
+    
+    func setupBackButton() {
+        backButton.setImage(UIImage(named: "BackArrow"), for: .normal)
+        backButton.addTarget(self, action: #selector(prevVC), for: .touchUpInside)
+        self.navigationController?.navigationBar.addSubview(backButton)
+        backButton.snp.makeConstraints { make in
+            make.leading.equalToSuperview().inset(25)
+            make.centerY.equalToSuperview()
+        }
+    }
+    
+    func setupTitle(name: String) {
+        self.navigationController?.navigationBar.titleTextAttributes = [.font: UIFont(name: "Sen-Regular", size: 24)]
+        self.navigationItem.title = name
+    }
+    
+    func setupTitleLines() {
+        let dottedLineMultiplier = 0.52
+        let solidLineVerticalInset = -12.75
+        let solidLineMultiplier = 0.32
+        let screenSize = UIScreen.main.bounds
+        let dottedline = UIImageView(image: UIImage(named: "dottedline"))
+        
+        dottedline.contentMode = .scaleAspectFit
+        view.addSubview(dottedline)
+        dottedline.snp.makeConstraints { make in
+            make.height.equalTo(1)
+            make.width.equalTo(screenSize.width*dottedLineMultiplier)
+            make.trailing.equalToSuperview().inset(10)
+            make.top.equalTo(view.safeAreaLayoutGuide)
+        }
+        
+        let solidline = UILabel()
```

Comment:
I don't think this needs to be a label. If you aren't adding any text, you can just use a `UIView`